images/router/haproxy/Dockerfile.rhel8

```diff
@@ -1,10 +1,10 @@
-FROM registry.ci.openshift.org/ocp/4.13:haproxy-router-base
+FROM registry.ci.openshift.org/ocp/4.13:base-router
```

Comment:
The actual "base" router image is named `base-router` when built from the Makefile, so I'm not sure where the `haproxy-router-base` tag comes from.

(Same as with the Makefile, I don't mind reverting this and coming back to it in another PR)